docs/usage/run.md

```diff
@@ -54,12 +54,23 @@ Each of the following arguments are required. Failure to provide a required argu
 > 
 > ***Example:*** `--output /data/$USER/WES_hg38`
 
+---  
+  `--runmode  {init,dryrun,run}`  `
+> **Execution Process.**  
+> *type: string*
+>   
+> User should initialize the pipeline folder by first running `--runmode init`  
+> User should then perform a dry-run to list all steps the pipeline will take`--runmode dryrun`  
+> User should then perform the full run `--runmode run`  
+> 
+> ***Example:*** `--runmode init` *THEN* `--runmode dryrun` *THEN* `--runmode run`
+
 ---  
   `--genome {hg38, custom.json}`
 > **Reference genome.**   
 > *type: string/file*
 >   
-> This option defines the reference genome for your set of samples. On Biowulf, exome-seek does comes bundled with pre built reference files for human samples; however, it is worth noting that the pipeline does accept a pre-built resource bundle pulled with the cache sub command (coming soon). Currently, the pipeline only supports the human reference hg38; however, support for mouse reference mm10 will be added soon.
+> This option defines the reference genome for your set of samples. On Biowulf, xavier does comes bundled with pre built reference files for human samples; however, it is worth noting that the pipeline does accept a pre-built resource bundle pulled with the cache sub command (coming soon). Currently, the pipeline only supports the human reference hg38; however, support for mouse reference mm10 will be added soon.
 >
 > ***Pre built Option***  
 > Here is a list of available pre built genomes on Biowulf: hg38.
```

Comment:
@samarth8392 This "pre built genomes" is pretty small! .I think we need mm10 (and hg19) as well. Please, we also need to be able replace "on Biowulf" with "on Biowulf and FRCE". Do you have some rough documentation on how to create these "pre built genomes" support? May be create a set of script(s) to automatically handle this?? @dnousome  any thoughts?